### PR TITLE
Turned exercise 008 into practice

### PR DESCRIPTION
**Pull request by komutumu**
Addresses issue #2176 
In this PR Exercise 008 has been turned into practice. 

 @cuihtlauac please review. 
 Thanks

### CODE REVIEW

**Review by cuihtlauac — CHANGES_REQUESTED** [review]
Please make sure the last character of each file is newline

**Review by christinerose — APPROVED** [review]

**Comment by komutumu** [comment]
> Please make sure the last character of each file is newline

Yes @cuihtlauac 
I have updated the PR please review again

File: practice/008/answer/impl.ml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+let rec compress = function
+    | a :: (b :: _ as t) -> if a = b then compress t else a :: compress t
+    | smaller -> smaller
+
+
```
Comment (cuihtlauac) [inline practice/008/answer/impl.ml]:
```suggestion
    | smaller -> smaller

```

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

**Review by cuihtlauac — APPROVED** [review]
Thanks @jahielkomu

**Comment by divyankachaudhari** [comment]
@jahielkomu @cuihtlauac This doesn't compile

```
~/ocaml.org/practice/008 (main ✘)✭ ᐅ dune build --root .
File "ex.ml", lines 8-17, characters 55-3:
 8 | .......................................................struct
 9 |   let tests = "compress" >::: [
10 |     "non-empty list" >:: (fun _ ->
11 |       assert_equal ["a"; "b"; "c"; "a"; "d"; "e"]
12 |         (Tested.compress ["a"; "a"; "a"; "a"; "b"; "c"; "c"; "a"; "a"; "d"; "e"; "e"; "e"; "e"]));
13 |     "empty list" >:: (fun _ -> assert_equal [] (Tested.compress []));
14 |   ]
15 | 
16 |   let tests = "Remove Duplicates" >::: [ tests ]
17 | end
Error: Signature mismatch:
       Modules do not match:
         sig val tests : test end
       is not included in
         sig val v : test end
       The value `v' is required but not provided
       File "ex.ml", line 8, characters 36-48: Expected declaration
 ```

**Comment by komutumu** [comment]
@divyankachaudhari you are absolutely right. 
I see why it isn't compiling. 

Is it okay if I update it?
Can you reopen the PR?

**Pull request merged**